.github/workflows/python-ci.yml

```diff
@@ -13,11 +13,14 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [ "3.8", "3.9", "3.10", "3.11", "3.12" ]
+        python-version: [ "3.8", "3.9", "3.10", "3.11", "3.12", "3.13" ]
```

Comment:
`3.8` is also EOL'd earlier this month, should we remove that too?

https://devguide.python.org/versions/

Reply:
follow-up PR OK for this?